draft-ietf-rats-architecture.md

```diff
@@ -842,40 +842,47 @@ a different format.
+The goal is merely to narrow their recentness to something the Verifier (for Evidence) or Relying Party (for Attestation Result) is willing to accept.
+Freshness is a key component  for enabling caching and reuse of both Evidence and Attestation Results, which is especially valuable in cases where their computation uses a substantial part of the resource budget (e.g., energy in constrained devices).
+
+There are two common approaches for determining the epoch of an Evidence or Attestation Result.
+
+<!-- Explicit Timekeeping using Synchronized Clocks -->
+
+The first approach is to rely on synchronized and trustworthy clocks, and include a signed timestamp (see {{?I-D.birkholz-rats-tuda}}) along with the Claims in the Evidence or Attestation Result.
+Timestamps can also be added on a per-Claim basis if wanted, to allow distinguishing the time of creation of Evidence from the time that a specific Claim value was generated.
+Clocks trustworthiness typically requires additional Claims about the signer's time synchronization mechanism.
+
+<!-- Implicit Timekeeping using Nonces -->
+
+A second approach places the onus of timekeeping solely on the appraising entity (e.g., the Verifier for Evidence, or the Relying Party for an Attestation Result).
+This might be an appropriate choice in case the Attester does not have a reliable clock or time synchronisation is otherwise impaired.
+In this approach, a non-predictable nonce is generated by the appraising entity, and the nonce is then signed and included along with the Claims in the Evidence or Attestation Result.
+After checking the sent and received nonces are the same, the appraising entity knows that the Claims were signed after the nonce was generated, effectively locking its clock to the produced payload.
+This allows associating a "rough" epoch to the Evidence or Attestation Result.
+In this case the epoch is said to be rough because:
+
+* it is always bound to the set of Claims as a whole, as opposed to having Claim being separately annotated, and
+* it is not possible to distinguish between time of collection from the time of creation of the Claims set.
+
+The two basic mechanisms presented above can be extended and combined in more advanced schemes.
+For example, if the clocks are trustworthy but not synchronized, first a nonce-based exchange may be used to determine the (rough) time offset between the involved peers, followed by any number of timestamp based exchanges.
+In another scenario where a broadcast channel is shared by all the RATS entities (Attesters, Verifiers and Relying Parties), the nonce-based approach may be used to lock all parties to the same timeline without requiring synchronized clocks by having a central entity emit nonces at regular intervals and have the "current" nonce included in the produced Evidence or Attestation Result.
```

Comment:
"Second, a broadcast channel shared by all Roles (Attester, Verifier, etc.), the nonce-based approach..."

Reply:
Not sure I understand this suggestion.  If the current text isn't good as-is, we may simplify it as "When a broadcast channel is shared by all Roles, the nonce-base approach [...]"

Reply:
Trying to get the signposting to match up. In line 880 signposting uses "first" and in 881 signposting changes to "another". Normally, "second" follows "first" - it is just style and I don't feel strongly about it.

Reply:
Line 880 and 881 are two separate examples.

The example at line 880 is split in two phases: first [...], followed by [...].

I will remove "first" since it isn't necessary and I suspect it's probably bad English if it got you confused.

Reply:
Ned: please check the proposal in 20e3d7